akka-docs/rst/java/code/docs/actor/ActorDocTest.java

```diff
@@ -11,6 +11,8 @@
 import akka.testkit.TestEvent;
 import com.typesafe.config.Config;
 import com.typesafe.config.ConfigFactory;
+import docs.AbstractJavaTest;
+import org.scalatest.junit.JUnitSuite;
```

Comment:
My train of thought was that each of them are a java-doc-test-suite in our project, and that a java-doc-test-suite needs to be a `JUnitSuite` is a technical detail, so expressing that in one place rather than all the places felt neater. Could probably have a better name and perhaps some docs though.

I don't have strong feelings about it though, so if you disagree I have no problem changing it to extending `JUnitSuite` directly.